source/index.js

```diff
@@ -13,12 +14,43 @@ renderer.roundPixels = true
 document.getElementById("mount").appendChild(renderer.view)
 window.game = new Pixi.Container()
 game.addChild(new Junkership())
+game.score=0
 
 game.addChild(new Trashbot(Reference.GAME_WIDTH, Reference.GAME_HEIGHT/2))
 
 var loop = new Afloop(function(delta) {
     game.children.forEach((child) => {
         child.update(delta)
+
+        if (child instanceof Trashbot ||
+            child instanceof Junkership){
+            collisionCheck(child)
+        }
```

Comment:
Certainly changeable if that's the consensus. I'd probably move the logic to the Junkership and Trashbot and not have the Projectiles check at all. Junkership checks for Projectile collisions (and then checks to make sure it didn't come from another Junkership) and Trashbot collisions, Trashbots just check for Projectiles. Which actually prompts the question of whether or not Trashbots have friendly fire on. Could, say, a Sniper accidentally take out a Snake if the player were positioned conveniently?

Let me know, fellas. Keep it or move it?